src/ILCompiler.TypeSystem/tests/project.json

```diff
@@ -2,7 +2,8 @@
   "dependencies": {
     "NETStandard.Library": "1.5.0-rc2-23911",
     "System.Reflection.Metadata": "1.3.0-rc2-23911",
-    "xunit": "2.1.0"
+    "xunit": "2.1.0",
+    "xunit.netcore.extensions": "1.0.0-prerelease-00231-07"
```

Comment:
This enables the usage of the [ActiveIssue(..)] custom attribute so that failing tests don't break the build.